charts/pega/templates/pega-environment-config.yaml

```diff
@@ -93,6 +93,14 @@ data:
 {{- else }}
   # URL to connect to Elastic Search
   PEGA_SEARCH_URL: {{ include "pegaSearchURL" $ }}
+{{- end }}
+{{- if and (.Values.pegasearch.externalSearchService) ((.Values.pegasearch.srsMTLS).enabled) }}
+  # To enable Mutual Transport Layer Security(MTLS) between Pega Platform and Search and Reporting Service(SRS). The default value is 'false'.
+  SRS_MTLS_ENABLED: "{{ .Values.pegasearch.srsMTLS.enabled }}"
```

Comment:
where does this setting used? I could not see its reference in pega-web-ready image

Reply:
@kishorv10  It is used in Platform source code to identify whether  mtls is enabled or not.